# Rate in runLoop()
loop_rate: 2000

# Step interval to publish in runLoop()
publish_interval: 20

# Height of xy plane marker
svm_thre: -0.1

# Limit of configuration update in one step [m], [rad]
delta_config_limit: 0.05

# Initial sample pose
initial_sample_pose:
  translation: [0, 0.5, 0]

# Whether to predict on grid map
grid_map_prediction: false

# Margin ratio of grid map
grid_map_margin_ratio: 0.0

# Resolution of grid map [m]
grid_map_resolution: 0.1

# Height scale of grid map
grid_map_height_scale: 1.0
